modules/integration/tests-integration/bpmn/src/test/java/org/wso2/bps/integration/tests/bpmn/BPMNCappDeploymentTestCase.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright (c) WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wso2.bps.integration.tests.bpmn;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.wso2.bps.integration.common.clients.bpmn.WorkflowServiceClient;
+import org.wso2.carbon.automation.engine.frameworkutils.FrameworkPathUtil;
+import org.wso2.carbon.integration.common.admin.client.CarbonAppUploaderClient;
+import org.testng.Assert;
+
+import java.io.File;
+import java.net.URL;
+import java.util.Calendar;
+import java.util.concurrent.TimeUnit;
+
+import org.wso2.carbon.integration.common.admin.client.ApplicationAdminClient;
+import org.wso2.bps.integration.common.utils.BPSMasterTest;
+
+import javax.activation.DataHandler;
+
+
+public class BPMNCappDeploymentTestCase extends BPSMasterTest {
+
+    private static final Log log = LogFactory.getLog(BPMNCappDeploymentTestCase.class);
+    String carFileName = "TestPro_1.0.0";
+    private boolean BPMNProcessExists = false;
+    private final int MAX_TIME = 120000;
+    private boolean isCarFileUploaded = false;
+    private ApplicationAdminClient applicationAdminClient;
+    private WorkflowServiceClient workflowServiceClient;
+
+    /**
+     * Deploy the uploaded C-App file
+     */
+
+    @BeforeClass(alwaysRun = true)
+    public void deployTask() throws Exception {
```

Comment:
Hi Imesh,
deployTask() method uses super.init() method which throws Exception object. Hence it forces to throw the super exception in deployTask().